xamarin/Xamarin.Forms
# [Enhancement] Label Autofit Font ✨

### Description of Change ###

I've created the `AutoFitText` (enum) property at `Label`, applying `AutoSizeTextType` flag on android and  `AdjustsFontSizeToFitWidth` on IOS

This new property can be used:

`AutoFitTextMode.None` -> Current Xamarin.Forms Label behavior
`AutoFitTextMode.FitToWidth` -> Label text will be resized (font) to fit control width

### Issues Resolved ### 
- fixes #7176

### API Changes ###

Added:
 - (enum) AutoFitTextMode Label.AutofitText{ get; set; } //Bindable Property
 - double Label.MaxAutoFitSize { get; set;} //Bindable Property
 - double Label.MinAutoFitSize {get; set;} //Bindable Property

### Platforms Affected ### 
- Core/XAML (all platforms)
- iOS
- Android

### Behavioral/Visual Changes ###
When `AutoFitText` == `AutoFitTextMode.FitToWidth` , label text will fill view width bounds, changing font size

### Before/After Screenshots ### 

- Before 
 None

- After
<p align="center">
	<kbd>
		<img src="https://user-images.githubusercontent.com/19656249/68448394-27215300-01c2-11ea-911b-6aa62db5e541.gif" alt="image" style="max-width:100%;"/>
	</kbd>
</p>


### Testing Procedure ###
1- Create a layout with some label inside;
2- Set Label `AutoFitText` property to `AutoFitTextMode.FitToWidth`;
3- Set Label `Text` property;
4- Text will fit the label;

> (See the gallery test 7176)

### PR Checklist ###
